doc/release-notes-14707.md

```diff
@@ -0,0 +1,17 @@
+Wallet `receivedby` RPCs now include coinbase transactions
+-------------
+
+Previously, the following wallet RPCs incorrectly filtered out coinbase transactions:
+
+`getreceivedbyaddress`
+
+`getreceivedbylabel`
+
+`listreceivedbyaddress`
+
+`listreceivedbylabel`
+
+This release corrects this behaviour and returns results accounting for received coins from coinbase outputs.
+
+A new option, `include_immature` (default=`false`), determines whether to account for immature coinbase transactions.
+Immature coinbase transactions are coinbase transactions that have 100 or fewer confirmations, and are not spendable.
```

Comment:
I think this should also document the `deprecatedrpc=filter_coinbase` (or `deprecatedrpc=exclude_coinbase` if you rename it) config option, including that the intention is to remove it after some time.